doc/nrf/gs_installing.rst

```diff
@@ -567,6 +578,8 @@ To install |SES| Nordic Edition, complete the following steps:
 
 .. _build_environment:
 
+.. rst-class:: numbered-step
+
 Setting up the build environment
 ********************************
 
```

Comment:
More step subsections below ... :(

Reply:
Need to think a second about how to handle subsections in this section, which is also not exactly the same in GSA.

Reply:
@ru-fu , please have another look. I've flattened out the structure of the page, so that there are no subsections in the main installation sections. I also moved the updating topics to the very end and moved the "setting up the command line environment" from subsection to an optional section after the SES one (command line sections is not present in GSA, so I think this is the way to go). But I have two doubts:
- I'm not convinced about the Updating section at the end -- Can it exist there? Would it make sense to make a separate page out of it?
- One of the links in Installing the toolchain points to the command line section, which is now optionally placed at the end, which breaks the flow -- Should we make this section mandatory and merge it with the SES one? Or rework the instruction that includes the link, so that we don't have to link at all?

Reply:
Discussed during NCS tech writer sync on Apr 1. Tracking task for creating an Updating page within Getting started: https://projecttools.nordicsemi.no/jira/browse/NCSDK-8936